stable/spinnaker/values.yaml

```diff
@@ -31,14 +31,14 @@ mail:
 
 # Images for each component
 images:
```

Comment:
I can't really control that they changed repos and version schemes in the [project at 1.0](https://www.spinnaker.io/community/releases/versions/). The config is all the same and the delta between the actual software versions can be measured in weeks and is backwards compatible.